### ISSUE TITLE
Content Resources (§5.7) example is unclear

### ISSUE THREAD

**Issue by ahankinson** [IIIF/api#1769]
In the 3.0 documentation the [guidance on Content Resources](https://preview.iiif.io/api/image-prezi-rc2/api/presentation/3.0/#57-content-resources) is unclear.

The example gives a `body` with an `id` of `"https://example.org/iiif/book1/res/page1.jpg"`. It then has a `service` block with an `id` of `"https://example.org/images/book1-page1"`, given as a IIIF Image Service 3 at level 2 compliance.

The specific things that are unclear in this example:

1. The relationship of '/iiif/book1/res/page1.jpg' to the image service. Given that the actual method of access of this JPEG content would be more along the lines of `/images/book1-page1/full/full/0/default.jpg`, it is unclear what relationship the Body `id` has with the Service `id`.

This is especially confusing since level 2 compliance almost certainly requires an image server, so it is unlikely that a request for `/iiif/book1/res/page1.jpg` will produce the desired result in the real world (a response of `image/jpeg`), and it will actually require a request to `/images/book1-page1/full/full/0/default.jpg` to *actually* retrieve a JPEG result of the page.

2. If, as many implementers do, the Body `id` can point to the image's `info.json` response and not to the actual image, does this mean that `type` on the Body can be something other than `Image`, and that `format` could be `application/ld+json`?

Speaking in specifics, as an implementer, the guidelines are quite ambiguous whether the Body `id`  should be:

`https://iiif.bodleian.ox.ac.uk/iiif/image/e58b8c60-005c-4c41-a22f-07d49cb25ede`

or 

`https://iiif.bodleian.ox.ac.uk/iiif/image/e58b8c60-005c-4c41-a22f-07d49cb25ede/full/full/0/default.jpg`

As a result, I might expect the example to look more along the lines of:

```
{
  "@context": "http://iiif.io/api/presentation/3/context.json",
  "id": "https://example.org/iiif/book1/annotation/p0001-image",
  "type": "Annotation",
  "motivation": "painting",
  "body": {
    "id": "https://example.org/images/book1-page1/full/full/0/default.jpg",
    "type": "Image",
    "label": {"en": ["Page 1"], "es": ["Página 1"]},
    "format": "image/jpeg",
    "service": [
      {
        "id": "https://example.org/images/book1-page1/info.json",
        "type": "ImageService3",
        "profile": "level2"
      }
    ],
    "height": 2000,
    "width": 1500
  },
  "target": "https://example.org/iiif/book1/canvas/p1"
}
```

With an additional clarification that it *could* also look like this:

```
 "body": {
    "id": "https://example.org/images/book1-page1",
    "type": "ImageService3",
    "label": {"en": ["Page 1"], "es": ["Página 1"]},
    "format": "application/ld+json",
    "service": [
    ...
```

...or a clarification that the `id` of the `body` MUST resolve to a result that reflects the content type of the resource (currently it's a SHOULD, which leaves it open to interpretation, and to the idea that you could return an info.json response for the content type of `image/jpeg`, and thus more confusion.)

Personally, I would rather that the Body `id` is de-referenceable to a JSON response and not to the image contents (i.e., `image/jpeg`). The reason is that it is common for the `full` image parameter to not provide an actual `full` image response that matches the dimensions of the canvas, but an image service *can* provide that. So a request for:

`https://iiif.bodleian.ox.ac.uk/iiif/image/e58b8c60-005c-4c41-a22f-07d49cb25ede/full/full/0/default.jpg`

will respond with an image that has 1000px maximum in any given dimension, but a request for any arbitrary segment of the image below the 1000px single-request limit will be drawn from the full-size image, making it possible for image viewers to construct the request for the full-size image from the `id` value of the `body`.

**Comment by azaroth42** [comment]
Editors agree that 1 is confusing as an example, and it should be changed.

However the examples in 2 are not valid. Will extract to a new issue as to how to deal with it.

**Comment by azaroth42** [comment]
Closed by #1802
